### PR TITLE
Add GitHub link and remove users route

### PR DESCRIPTION
**Pull request by sararob**
@tstirrat some minor updates to the dummy app. Could you review and merge?

### CODE REVIEW

**Comment by tstirrat** [comment]
Let's add the view source link to to global header so it's always visible. and maybe something like the github fork things in the left corner here: https://firefeed.io/

**Comment by sararob** [comment]
@tstirrat updated. Could you take a look?

File: tests/dummy/app/templates/application.hbs [hunk 1]
```diff
@@ -5,6 +5,7 @@
   <nav>
     {{#link-to "auth"}}Try Authentication{{/link-to}}
     {{#link-to "posts.new"}}New Post{{/link-to}}
+    <a href="https://github.com/firebase/emberfire/tree/master/tests/dummy"><img style="position: absolute; top: 0; right: 0; border: 0;" src="https://camo.githubusercontent.com/a6677b08c955af8400f44c6298f40e7d19cc5b2d/68747470733a2f2f73332e616d617a6f6e6177732e636f6d2f6769746875622f726962626f6e732f666f726b6d655f72696768745f677261795f3664366436642e706e67" alt="Fork me on GitHub" data-canonical-src="https://s3.amazonaws.com/github/ribbons/forkme_right_gray_6d6d6d.png"></a>
```
Comment (tstirrat) [inline tests/dummy/app/templates/application.hbs]: remove data-canonical-src and change `src

**Comment by tstirrat** [comment]
This also needs a rebase and squash

**Pull request merged**